docs/synapse/devopsguide.rst

```diff
@@ -1102,6 +1102,59 @@ additonal CA files.
     is similar to the Cortex, but uses the ``SYN_AXON_TLS_CA_DIR`` environment
     variable.
 
+Axon Blob Export and Import
+----------------------------
+
+For situations where an organization needs to synchronize Axon blobs between two Axons
+that do not connect directly to each other, due to network segmentation as an example,
+there are command line tools available to export blobs from a source Axon and then import
+those blobs into a different Axon. For Axons that can communicate directly, the
+``axon2axon`` tool should be used for live synchronization.
+
+- ``synapse.tools.axon.dump``: Export blobs from a specified Axon.
+- ``synapse.tools.axon.load``: Import blobs to a specified Axon.
+
+In these situations, an export would be created using
+``synapse.tools.axon.dump``. The output of this tool is one or more export
+files that can then be manually copied over to the destination Axon
+environment. Once the files have been copied, the ``synapse.tools.axon.load``
+tool can be used to import them into the destination Axon.
+
+.. note::
+
+   The ``synapse.tools.axon.dump`` export tool can create a state tracking YAML
+   file for recording the next expected offset from the Axon that was exported.
+   The state tracking file allows the export tool to create incremental exports
+   from the time of the last execution without requiring the offset to be
+   separately tracked.
+
+   The state tracking file can be created and used by the ``synapse.tools.axon.load``
+   tool using the ``--statefile`` command line option with a valid path. The state
+   tracking file is saved in that directory with name ``<celliden>.yaml``.
```

Comment:
This is confusing. The name `--statefile` makes me think this argument is a path to a file, not a directory (which is how it is in the layer tools). Maybe it should be `--statedir`? That would be different than the layer tools though so I'd recommend updating this behavior to be a path to a file.